src/LabelSelector/LabelChip/LabelChip.stories.tsx

```diff
@@ -79,3 +79,21 @@ export const Small = {
 
   render: Template
 };
+
+export const Selected = {
+  args: {
+    ...Default.args,
+    selected: true
+  },
+
+  render: Template
+};
+
+export const Tooltip = {
```

Comment:
Missing code doc describing the intention of the story. What use case are we showing the developer with these stories.